src/Groups/matrices/MatGrp.jl

```diff
@@ -498,8 +498,6 @@ function gens(G::MatrixGroup)
    return G.gens
 end
 
-gen(G::MatrixGroup, i::Int) = gens(G)[i]
```

Comment:
I think we had that because the matrix groups cache the generators, and the generators here are more than just wrappers around the GAP group elements; they can also (optionally) contain Julia matrices (this should have been explained here in a comment...)